src/Mailgun/Connection/RestClient.php

```diff
@@ -147,6 +147,9 @@ public function post($endpointUrl, array $postData = [], $files = [])
                     $postDataMultipart[] = [
                         'name' => sprintf('%s[%d]', $key, $index++),
                         'contents' => $subValue,
+                        'headers' => [
+                            'Content-Disposition' => sprintf('form-data; name="%s"', $key),
```

Comment:
Im not sure. I do not work with the Mailgun api team. (I do not work at Mailgun at all..) I just know it is good practice to use the array syntax instead. **All** HTTP servers and server languages will support the array syntax.